docs/spec/pubsub-protocol-binding.md

```diff
@@ -153,7 +156,7 @@ Examples:
     * `id` maps to `ce-id`
     * `specversion` maps to `ce-specversion`
 
-Attribute names are case-insensitive.
+Attribute names are lower-case.
```

Comment:
If all attribute names are lower case, it seems a bit odd to use `contentType` as the only attribute with an uppercase character.

Reply:
changed it!